### DISCUSSION TITLE
Revise "confirmation rate" language in the network indicator

### DISCUSSION THREAD

**Discussion by saurabhtukul** [iotaledger/firefly#1550] [Features / Enhancements]
## Description

Used firefly beta wallet.
Its shows 101 % confirmation rate which in any case could not be true.

The calculation for confirmation rate is faulty, it cannot be more than 100%  whatever may be the criteria, which i assume is part of wallet.rs and firefly just shows the data.
## Proof of Concept

download firefly beta wallet and create a new wallet and click on network status 

![Screenshot 2021-03-23 at 6 38 29 AM](https://user-images.githubusercontent.com/7590517/112080925-d177fb80-8ba8-11eb-9cb0-3aa5dbfff82e.png)

## Open feedback (optional)

Use this section to add any feedback related to the submission:

**Comment by saurabhtukul** [comment]
It will basically lead to faulty expectation from network when the load is too high...

**Comment by Thoralf-M** [comment]
The confirmation rate is calculated in the explorer and is "wrong" because the current mps can change and there can be messages parallel to a milestone in the tangle. At one time a milestone might confirm not all other messages that are there at this moment, because they are sent parallel to the milestone in the tangle and the next time it could reference also the messages that were parallel the previous time, then the milestone confirms more messages than were sent during the time from the last milestone and thus the confirmation rate is over 100%

**Comment by saurabhtukul** [comment]
If at current point ,it could shows wrong value, better not to show it...Its very ambiguous to user seeing more than 100% confirmation whatever the underlaying parallel universe, it should and must not cross 100% otherwise it basically depicts a flux state where consensus is not uniform.

**Comment by Thoralf-M** [comment]
It's not wrong, in this timeframe more messages got confirmed than sent

**Comment by saurabhtukul** [comment]
Then the definition of confirmation is wrong.You are confusing user...
Confirmation should only mean if all the transaction which is supposed to get confirmed in 10 sec(if thats the benchmark now) is confirmed or not.It will show 100% only if all those transaction which are meant to be confirmed within 10 secs are success-full.
Otherwise it has to be less than 100%..

**Comment by maxwellmattryan** [comment]
@saurabhtukul So would you propose just hard-limiting the rate (in Firefly) at 100% as to not confuse the user? Another alternative I can think of is a tooltip explaining this, however I'm leaning towards hard-limiting at 100% personally.

**Comment by rafaeldjpbrochado** [comment]
I agree this is an issue. Here's another discussion about it: https://github.com/iotaledger/firefly/discussions/1532
